nasa/earthdata-search
# EDSC-3647: fix download script issue, where status code appended without line break

# Overview

Download script would fail randomly because curl was adding a status code at the end of a request without a line break. And thus, tail would see the status code and the previous data as one long line.

### What is the feature?

Basically tell curl's write-out feature to add a new line character before writing the status code

### What is the Solution?

I modified the code that generates the download script

### What areas of the application does this impact?

Download Script tab

# Testing

### Reproduction steps

- **Environment for testing:**
- **Collection to test with:**

1. go to the main page
2. search for C1748066515-LPCLOUD
3. click on the one search result
4. add the first granule to your project
5. go to the project
6. click download data
7. on the download page, click the Download Script tab
8. click save and save the download script to your computer
9. run the download script via bash (e.g. `bash 4517239960-download.sh`)
10. fill out your username and password when prompted
11. check your folder to see if the download worked

*note*: if you wanted to check to make sure there wasn't any regression, you could also try the steps above for C1996881752-POCLOUD

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
